app/components/joyride-foundation.js

```diff
@@ -43,6 +54,10 @@ export default Ember.Component.extend({
         function showJoyrideOverlay(){
           Ember.$(".joyride-view-background").show();
         }
+
+        function setSeenTour(){
+          _this.set("session.seenTour", _this.get("session.seenTour") + "," + _this.get("currentUserId"));
```

Comment:
so shall I change it to just verify the currentUser and not to keep list of other users? @steveyken